authzed/api/v1/error_reason.proto

```diff
@@ -285,4 +285,17 @@ enum ErrorReason {
   //       }
   //     }
   ERROR_REASON_MAXIMUM_DEPTH_EXCEEDED = 19;
+
+  // The request failed due to a serialization error in the backend database.
+  // This typically indicates that various in flight transactions conflicted with each other
+  // and the database had to abort one or more of them.
```

Comment:
Add a note that retries were attempted before the error was raised

Reply:
done